examples/Producer/Program.cs

```diff
@@ -93,8 +79,8 @@ public static void Main(string[] args)
                         break;
                     }
 
-                    var key = "";
-                    var val = text;
+                    string key = null;
```

Comment:
just curious: why is this better than type inference?

Reply:
type inference doesn't work with null and I explicitly specified `string` for val (which is a string) as it looked nicer to me next to using the same for key. but this is pretty arbitrary.